lib/pairmotron/pair_maker.ex

```diff
@@ -26,7 +26,7 @@ defmodule Pairmotron.PairMaker do
       |> Repo.all
   end
 
-  defp fetch_users_from_pairs(pairs) do
+  defp preload_users(pairs) do
```

Comment:
Come to think of it, I'm not sure why this was extracted. All it's doing is preloading users. Couldn't we just pipe to `Repo.preload([:users])` directly on line 10?